stl/inc/ostream

```diff
@@ -546,18 +546,24 @@ public:
         if (_Rdbuf) { // buffer exists, flush it
             const sentry _Ok(*this);
 
+            _TRY_IO_BEGIN
             if (_Ok && _Rdbuf->pubsync() == -1) {

```

Comment:
Should we move the `_Ok` check outside of the `try`/`catch` block? (I think this has a tiny perf impact on x86, and it would be consistent with `put` and `write` above.)

Reply:
I am slightly inclined to walk all over the file but maybe in a different PR